### PR TITLE
examples: Add example for multiple windows

### LINKED ISSUE #561: Support apps without root window
Many apps need the ability to open multiple windows and closing any window without closing the app. For example, a user would open multiple windows of a web browser and then close any opened window they choose while expecting others to stay open. Currently, Relm4 only allows closing windows that is not the main app window, and if the user closes the that the whole app closes. Supporting otherwise would allow Relm4 to be much more versatile and suitable for a much larger range of apps.

### PR DESCRIPTION
**Pull request by AaronErhardt**
#### Summary

Fixes #561

@kdwk

#### Checklist

- [x] cargo fmt
- [x] cargo clippy
- [x] cargo test
- [x] updated CHANGES.md

### CODE REVIEW

**Comment by LevitatingBusinessMan** [comment]
This seems to just be an exact copy of the `simple.rs` example?

**Comment by AaronErhardt** [comment]
95% oft the code is indeed identical, but there is a new message variant for creating new windows.

File: examples/multi_window.rs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                self.counter = self.counter.wrapping_add(1);
+            }
+            Msg::Decrement => {
+                self.counter = self.counter.wrapping_sub(1);
+            }
+            Msg::NewWindow => {
+                let app = relm4::main_application();
+                let builder = Self::builder();
+
+                app.add_window(&builder.root);
```
[the remark this answers is not in the document]
Comment (AaronErhardt) [reply examples/multi_window.rs:86]: Done :)

**Review by blabno — APPROVED** [review]
Great example!

**Pull request merged**